examples/monaco-graphql-webpack/src/index.ts

```diff
@@ -1,4 +1,7 @@
-import * as monaco from 'monaco-editor/esm/vs/editor/editor.api.js';
+// eslint-disable-next-line spaced-comment
+/// <reference path='../../../node_modules/monaco-editor/monaco.d.ts'/>
+// eslint-disable-next-line spaced-comment
+/// <reference path='../../../packages/monaco-graphql/src/typings/monaco.d.ts'/>
```

Comment:
it looks like theres a config to support this! https://github.com/typescript-eslint/typescript-eslint/issues/600

```
/* eslint spaced-comment: ["error", "always", { "markers": ["/"] }] */

/// This is a comment with a marker
```

Reply:
that would be great, this eslint fix has been quietly breaking our declarations left and right. hahaha!